.github/workflows/publish_pytorch_dev_docker.yml

```diff
@@ -71,4 +84,54 @@ jobs:
             AMDGPU_TARGETS=${{ matrix.targets.amdgpu_target }}
           push: true
           tags: ${{ steps.meta.outputs.tags }}
-          labels: ${{ steps.meta.outputs.labels }}
+          labels: |
+            ${{ steps.meta.outputs.labels }}
+            org.opencontainers.image.title="PyTorch ROCm Dev Image"
+            org.opencontainers.image.description="Nightly ROCm PyTorch Dev Docker for target ${{ matrix.targets.amdgpu_target }}""
+            org.opencontainers.image.version=${{ env.VERSION }}
+            org.opencontainers.image.created=${{ steps.meta.outputs.created }}
+            org.opencontainers.image.revision=${{ github.sha }}
+
+      - name: Log digest
+        run: |
+          echo "Built image with digest: ${{ steps.build_push.outputs.digest }}"
+
+      - name: Trigger verification workflow
+        uses: benc-uk/workflow-dispatch@e2e5e9a103e331dad343f381a29e654aea3cf8fc #1.2.4
+        with:
+          workflow: verify_docker_image.yml
+          inputs: |
+            {
+              "amdgpu_target": "${{ matrix.targets.amdgpu_target }}",
+              "image_digest": "${{ steps.build_push.outputs.digest }}",
+              "runner_type": "${{
+                matrix.targets.amdgpu_target == 'gfx942' && 'linux-mi300-1gpu-ossci-rocm' ||
+                (matrix.targets.amdgpu_target == 'gfx1201' && 'linux-rx9070-gpu-rocm') ||
+                (matrix.targets.amdgpu_target == 'gfx1100' && 'linux-rx7900-gpu-rocm') ||
+                'linux-mi300-1gpu-ossci-rocm'
+              }}",
+              "dispatch_id": "${{ env.DISPATCH_ID }}"
+            }
+
+      - name: Wait for verification to complete
+        run: |
+          external-builds/pytorch/wait_for_verification.sh
```

Comment:
What is the advantage of running a shell script here? You could either

* Move this to the `Pull and Verify Docker Image` job
* Move it to a separate job that gets dispatched from `Pull and Verify Docker Image` via `benc-uk/workflow-dispatch`.

Both would allow to drop installing additional tools and the need for a bash script.

Reply:
It is a long script that will make the workflow seem more complex and it is another phase a after  `Pull and Verify Docker Image job` (That is why I did not put this into `Pull and Verify Docker Image job`). The purpose of the script to find correct job and detect its status.

Reply:
Yeah, but if you dispatch a sub job you can drop the entire script as well as the logic. As you know which image is verified you also know which image gets verified. No need to find and check status.

Reply:
I think I got the idea - will implement that.